types/types.d.ts

```diff
@@ -53,3 +53,11 @@ declare interface Data {
 declare interface Options {
   [key: string]: any;
 }
+
```

Comment:
I think the `types/types.d.ts` file represents some generic types that are used in `foundry.js` It's basically the content of the comments in the first few lines of code of foundry.js. We probably should place our own utility types somewhere else. I don't know what the best place is, though.

Reply:
Yes that is exactly what this file is about. And I agree with ghost. We should put our own custom stuff to a distinct file, so it easier to keep them apart.

Reply:
Where should I put it then?

Reply:
How about we put it in a file like `types/typesUtils.d.ts`?